game-core/src/system/enemy.rs

```diff
@@ -3,25 +3,22 @@ use amethyst::{
     ecs::{Entities, Join, Read, ReadStorage, System, WriteStorage},
     renderer::{SpriteRender, Transparent},
 };
-use crate::component::Animation;
-use crate::component::Enemy;
-use crate::component::Player;
+use crate::component::{Animation, Enemy, Player};
 use rand::distributions::{Distribution, Uniform};
 
 pub struct Movement;
 
 impl<'s> System<'s> for Movement {
     type SystemData = (ReadStorage<'s, Enemy>, WriteStorage<'s, Transform>);
 
-    fn run(&mut self, (enemy, mut transforms): Self::SystemData) {
-        for (_, transform) in (&enemy, &mut transforms).join() {
-            println!("enemy: {:?}", transform);
-        }
+    fn run(&mut self, (players, mut transforms): Self::SystemData) {
```

Comment:
Can you explain to me why this is suppose to be players instead of enemy? I might be reading this wrong.

Reply:
It definitely shouldn't be, merge mistake. Nice catch.